test/stdlib/PrintFloat.swift.gyb

```diff
@@ -623,12 +623,20 @@ PrintTests.test("Printable_Float16") {
   expectNaN("nan(0xff)", Float16(bitPattern: 0x7eff))
   expectNaN("-nan", -Float16.nan)
   expectNaN("-nan(0xff)", -Float16(nan: 255, signaling: false))
-  expectFailure { expectNaN("snan", Float16.signalingNaN) }
-  expectFailure { expectNaN("-snan", -Float16.signalingNaN) }
-  expectFailure { expectNaN("snan(0xff)", Float16(nan: 255, signaling: true)) }
-  expectFailure { expectNaN("-snan(0xff)", -Float16(nan: 255, signaling: true)) }
-  expectFailure { expectNaN("snan(0xff)", Float16(bitPattern: 0x7dff)) }
-
+/*
+  // These fail on macOS x86_64, pass on iphonesimulator-i386, and
+  // probably behave in varying fashion on ARM32 and ARM64.
+  // So I'll just comment them out for now...
+  // Once we get real Float16 argument passing everywhere,
+  // these can be enabled again.
+  expectFailure {
+  expectNaN("snan", Float16.signalingNaN)
+  expectNaN("-snan", -Float16.signalingNaN)
+  expectNaN("snan(0xff)", Float16(nan: 255, signaling: true))
+  expectNaN("-snan(0xff)", -Float16(nan: 255, signaling: true))
+  expectNaN("snan(0xff)", Float16(bitPattern: 0x7dff))
+  }
+*/
   expectEqual("nan", Float16.signalingNaN.description)
```

Comment:
You might need to drop these as well. Can we just check that the result contains "nan"?

Reply:
The `description` property always returns the constant string "nan" for any NaN, ignoring payload and flags.  So the `expectEqual` checks here are correct.

Only the `debugDescription` contains any detail about nans.